packages/runtime/src/useSSRComputation_Server.tsx

```diff
@@ -9,12 +10,13 @@ export default function useSSRComputation_Server(fn: (...dependencies: any[]) =>
     // relativePathToCwd is used to make sure that the cache key is unique for each module
     // and it's not affected by the file that calls it
     const cacheKey = calculateCacheKey(relativePathToCwd, dependencies);
-    // check if result is a promise
-    if (result && typeof result.then === 'function') {
-      cache[cacheKey] = result.then(asyncResult => cache[cacheKey] = asyncResult);
```

Comment:
I added this line before to deal with promises. But, I don't know if server-side renderer will support executing async functions. Should I remove it?

Reply:
One of the coding principles I follow is "Every single line of code should be tested". If you don't have proof that the code works, don't merge it.

So I'm for reverting (unless you tested it and found it useful).

Reply:
@Romex91 I don't add lines here, I delete them. I removed the ability to track async results on the server-side. This feature is already not been tested before. So, I think it's a mistake from me to add it before.

Reply:
Yes, exactly. So I approve the reversion.